controller/user/json.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2018 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package user
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+
+	"gopkg.in/mgo.v2"
+)
+
+const (
+	adminDB = "admin"
+)
+
+type UserRoleJSON struct {
+	Role     string `json:"role"`
+	Database string `json:"db"`
+}
+
+type UserJSON struct {
```

Comment:
type name will be used as user.UserJSON by other packages, and that stutters; consider calling this JSON